// @devvai/devv-code-backend/dist/index.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
exports.email = exports.upload = exports.table = exports.auth = exports.DevvAI = exports.DevvEmail = exports.FileUpload = exports.ProjectTable = exports.DevvAuth = void 0;
var auth_1 = require("./auth");
Object.defineProperty(exports, "DevvAuth", { enumerable: true, get: function () { return auth_1.DevvAuth; } });
var table_1 = require("./table");
Object.defineProperty(exports, "ProjectTable", { enumerable: true, get: function () { return table_1.ProjectTable; } });
var upload_1 = require("./upload");
Object.defineProperty(exports, "FileUpload", { enumerable: true, get: function () { return upload_1.FileUpload; } });
var email_1 = require("./email");
Object.defineProperty(exports, "DevvEmail", { enumerable: true, get: function () { return email_1.DevvEmail; } });
var ai_1 = require("./ai");
Object.defineProperty(exports, "DevvAI", { enumerable: true, get: function () { return ai_1.DevvAI; } });
const auth_2 = require("./auth");
const table_2 = require("./table");
const upload_2 = require("./upload");
const email_2 = require("./email");
const device_1 = require("./device");
// Initialize device ID when SDK is first loaded
(0, device_1.initializeDeviceId)();
exports.auth = new auth_2.DevvAuth();
exports.table = new table_2.ProjectTable();
exports.upload = new upload_2.FileUpload();
exports.email = new email_2.DevvEmail();
